web/docs/deployment/deployment-methods/paas.md

```diff
@@ -663,8 +553,213 @@ If you wish to deploy an app leveraging [Jobs](../../advanced/jobs) that use `pg
 Read more: https://devcenter.heroku.com/articles/connecting-heroku-postgres#connecting-in-node-js
 :::
 
-## Koyeb (server, client and database)
+## Netlify <Client />
+
+Netlify is a static hosting solution that is free for many use cases. You will need a Netlify account to follow these instructions.
+
+Make sure you are logged in with Netlify CLI. You can check if you are logged in with `netlify status`, and if you are not, you can log in with `netlify login`.
+
+First, make sure you have [built the Wasp app](#1-generating-deployable-code). We'll build the client web app next.
+
+<BuildingTheWebClient />
+
+We can now deploy the client with:
+
+```shell
+npx netlify-cli deploy
+```
+
+<small>
+
+Carefully follow the instructions i.e. do you want to create a new app or use an existing one, the team under which your app will reside etc.
+
+</small>
+
+The final step is to run:
+
+```shell
+npx netlify-cli deploy --prod
+```
+
+That is it! Your client should be live at `https://<app-name>.netlify.app`.
+
+:::note
+Make sure you set this URL as the `WASP_WEB_CLIENT_URL` environment variable in your server hosting environment.
+:::
+
+:::caution Redirecting URLs toward `index.html`
+If you follow the instructions above, Netlify will use `netlify.toml` file that Wasp generates by default in `.wasp/build/web-app/`. This will correctly configure Netlify to redirect URLs toward `index.html`, which is important since Wasp is a Single Page Application (SPA) and needs to handle routing on the client side.
+
+If you instead use another method of deployment to Netlify, e.g. do it via CI, make sure that Netlify picks up that `netlify.toml` file, or configure URL redirecting yourself manually on Netlify.
+
+It is recommended to deploy through the Netlify CLI in Github Actions. The first deploy can be through the website or manually to get a `NETLIFY_SITE_ID`, the following deploys can then be automatic.
+:::
+
+### Deploying through Github Actions
+
+To enable automatic deployment of the client whenever you push to the `main` branch, you can set up a GitHub Actions workflow. To do this, create a file in your repository at `.github/workflows/deploy.yaml`. Feel free to rename `deploy.yaml` as long as the file type is not changed.
+
+Here’s an example configuration file to help you get started. This example workflow will trigger a deployment to Netlify whenever changes are pushed to the main branch.
+
+<details>
+<summary>Example Github Action</summary>
+
+```yaml
+name: Deploy Client to Netlify
```

Comment:
A lot of stuff in this Github Action is not Netlify specific, but is GA or Wasp installation/setup specific. That part is then also duplciated between this part of docs and the one below for CloudFlare.

Can we extract this common part? I would extract it into "CI" section of docs, have this general setup there (how to get going with CI setup for Wasp), and then here we would provide just GA steps that are specific for Netlify or for CloudFlare.